### PR TITLE
Complete Jira Oauth integration for test cases.

### PR DESCRIPTION
**Pull request by 99M8**

### CODE REVIEW

File: defaultConfig.json [hunk 1]
```diff
@@ -26,6 +26,9 @@
         "request-token": "/plugins/servlet/oauth/request-token",
         "access-token": "/plugins/servlet/oauth/access-token",
         "authorize": "/plugins/servlet/oauth/authorize"
+      },
+      "stories": {
```
Comment (JohnMBarry) [inline defaultConfig.json]: Remove if not required for this task.

File: tasks/_test.js [hunk 2]
```diff
@@ -99,11 +102,7 @@ module.exports = function testTasks(gulp, context) {
       }
     }, glob.sync(sourceGlobStr));
 
-    //set YADDA_FEATURE_GLOB if argv[2]
-    if (context.argv.length === 2) {
-      process.env.YADDA_FEATURE_GLOB = context.argv[1];
-      logger.info("Set process.env.YADDA_FEATURE_GLOB=" + process.env.YADDA_FEATURE_GLOB);
-    }
+
```
Comment (JohnMBarry) [inline tasks/_test.js]: Revert change

Comment (99M8) [reply tasks/_test.js]: Why?

File: defaultConfig.json (hunk 1 above)
Comment (99M8) [reply defaultConfig.json]: This is required (jiraConfig.stories.regex). Please review further changes in the _test.js file (line 426)

**Pull request merged**